styles/mixins/patterns/_tables.scss

```diff
@@ -42,10 +51,11 @@ $-min-row-height: var.$touch-target-size;
 
     th {
```

Comment:
There could be other `th` in the table. I would suggest to narrow down this style to `th` that are children of `thead` or apply this to `thead` if possible.

Reply:
I'd like this styling to apply to any `th` to a table in this pattern. If we find a counter example, we can update the styling! This won't leak out to non `.Hyp-Table` tables, so I think we're OK for now.